src/parcsr_mv/par_csr_matrix.c

```diff
@@ -1835,6 +1835,8 @@ hypre_CSRMatrixToParCSRMatrix( MPI_Comm         comm,
    HYPRE_Int          *num_nonzeros_proc;
    HYPRE_BigInt        row_starts[2];
    HYPRE_BigInt        col_starts[2];
+   HYPRE_Int           valid_row_starts = 0;
+   HYPRE_Int           valid_col_starts = 0;
```

Comment:
```suggestion
   HYPRE_BigInt        row_starts[2] = {0, 0};
   HYPRE_BigInt        col_starts[2] = {0, 0};
```

Maybe this solution is enough and doesn't require the addition of the `valid` variables

Reply:
Nevermind, this won't work. Sorry for the confusion